From: Antonio Valentino <antonio.valentino@tiscali.it>
Date: Sat, 5 Aug 2023 07:00:05 +0000
Subject: Compatibility with sphinx 7.1

Forwarded: not-needed
Origin: https://github.com/PyTables/PyTables/commit/72bedb052e89210b43a48048a837b4d960c3e1c6
---
 doc/source/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/source/conf.py b/doc/source/conf.py
index 0bdeec5..360328d 100644
--- a/doc/source/conf.py
+++ b/doc/source/conf.py
@@ -199,7 +199,7 @@ intersphinx_mapping = {'https://docs.python.org/': None}
 
 # -- External link options ----------------------------------------------------
 extlinks = {
-    'issue': ('https://github.com/PyTables/PyTables/issues/%s', 'gh-'),
+    'issue': ('https://github.com/PyTables/PyTables/issues/%s', 'gh-%s'),
 }
 
 # -- Options for autodocumentation ---------------------------------------------
